proto/ibc/core/channel/v1/channel.proto

```diff
@@ -174,6 +174,42 @@ message PacketId {
   uint64 sequence = 3;
 }
 
+// PacketV2 defines a type that carries data across different chains through IBC
+message PacketV2 {
+  // number corresponds to the order of sends and receives, where a Packet
+  // with an earlier sequence number must be sent and received before a Packet
+  // with a later sequence number.
+  uint64 sequence = 1;
+  // identifies the sending chain.
+  string source_id = 2;
+  // identifies the receiving chain.
+  string destination_id = 3;
+  // timeout after which the packet times out
+  string timeout = 4;
```

Comment:
not convinced this is the best idea, @damiannolan also pointed out that string parsing is expensive in an EVM environment. 

Is there a good reason not to just use uint64 nanos only?

cc @AdityaSripal

Reply:
I also agree that we should just have uint64